src/app/wallets/ln-send-payment.ts

```diff
@@ -481,9 +491,18 @@ const executePaymentViaLn = async ({
           milliSatsAmount: toMilliSatsFromNumber(amount * 1000),
           maxFee,
         })
+
+    const payment = await lndService.lookupPayment({ pubkey, paymentHash })
+    if (payment instanceof Error) return payment
+
+    payment.paymentRequest = payment.paymentRequest || paymentRequest
+    payment.paymentHash = payment.paymentHash || paymentHash
+    const persistedPayment = await LnPaymentsRepository().update(payment)
+    if (persistedPayment instanceof Error) return persistedPayment
+
     if (payResult instanceof LnPaymentPendingError) return PaymentSendStatus.Pending
 
-    const settled = await ledgerService.settlePendingLnPayments(paymentHash)
+    const settled = await ledgerService.settlePendingLnPayments({ paymentHash })
```

Comment:
Very possible I'm missing something here but I think we decided to drop the braces for single arguments.